entrypoint

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
```

Comment:
Add License and Copyright.

Reply:
Added MIT License and Copyright (c) 2022 Foundries.io